### PR TITLE
Add Rating in Component library

### PR DESCRIPTION
**Pull request by nikhilmalviya-dev**
Add different types of Rating in component library

### CODE REVIEW

File: Rating/Rating.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rating</title>
+    <link rel="stylesheet" href="Rating.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="header">
```
Comment (adith-a-danthi) [inline Rating/Rating.html:12]: Try to use semantic tag `header`

File: Rating/Rating.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        <div class="home">
+            <img src="logo.png " width="50px" height="50px" alt="">
+            <a class="home-link" href="/">Home</a>
+        </div>
+    </div>
+    <div class="container">
```
Comment (adith-a-danthi) [inline Rating/Rating.html:18]: You can use semantic tag `main` instead of `div`

File: Rating/Rating.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    --special-margin-top:1rem;
+    --font-size:1.5rem;
+}
+.code{
+    margin-top: 2rem;
+}
+.header{
+    display: flex;
+    justify-content: space-between;
+    background-color:rgb(139, 185, 179);
```
Comment (adith-a-danthi) [inline Rating/Rating.css:14]: You can create a CSS variable for the color if it is being used in other places as well

**Review by adith-a-danthi — CHANGES_REQUESTED** [review]
Looks Good! Try using semantic HTML tags like `header`, `main`.

File: Rating/Rating.html (hunk 1 above)
Comment (nikhilmalviya-dev) [reply Rating/Rating.html:12]: ok will do that

**Review by Bharatrathod-81 — APPROVED** [review]
good work

**Pull request merged**